.github/workflows/ci_cd.yml

```diff
@@ -152,30 +142,28 @@ jobs:
           ANSYSLMD_LICENSE_FILE: ${{ format('1055@{0}', secrets.LICENSE_SERVER) }}
           PYFLUENT_START_INSTANCE: 0
           PYFLUENT_LAUNCH_CONTAINER: 1
-          DOCS_CNAME: fluentparametric.docs.pyansys.com
+          DOCS_CNAME: parametric.fluent.docs.pyansys.com
```

Comment:
This line can be removed in favor of the environment variable declared at the top of the file:

```suggestion
```

Make sure that this `DOCS_CNAME` is replaced if any other occurrences are present.